src/scss/components/ec-btn/ec-btn.story.js

```diff
@@ -0,0 +1,163 @@
+import { storiesOf } from '@storybook/vue';
+import { text, boolean } from '@storybook/addon-knobs';
+import StoryRouter from 'storybook-vue-router';
+import EcIcon from '@/components/ec-icon';
+
+const stories = storiesOf('Button', module);
+
+
+stories
+  .addDecorator(StoryRouter())
+  .add('basic', () => ({
+    components: { EcIcon },
+    props: {
+      btnclass: {
+        default: text('class', 'ec-btn ec-btn--primary'),
+      },
+      isDisabled: {
+        default: boolean('Disabled', false),
+      },
+      hasIcon: {
+        default: boolean('Has Icon', false),
+      },
+      isSmall: {
+        default: boolean('ec-btn--sm', false),
+      },
+      isMedium: {
+        default: boolean('ec-btn--md', false),
+      },
+      isRounded: {
+        default: boolean('ec-btn--rounded', false),
+      },
+      isNegative: {
+        default: boolean('ec-btn--negative', false),
+      },
+      isPrimary: {
+        default: boolean('ec-btn--primary', false),
+      },
+      isSecondary: {
+        default: boolean('ec-btn--secondary', false),
+      },
+      isSuccess: {
+        default: boolean('ec-btn--success', false),
+      },
+      isError: {
+        default: boolean('ec-btn--error', false),
+      },
+    },
+    template: `
+      <div style="margin:20px;">
+        <div>
+          <h3 class="ec--m10">This is a button tag with a "ec-btn" class</h3>
+          <button
+            :class="{
+              'ec-btn--primary' : isPrimary,
+              'ec-btn--secondary' : isSecondary,
+              'ec-btn--success' : isSuccess,
+              'ec-btn--error' : isError,
+              'ec-btn--rounded' : isRounded,
+              'ec-btn--negative' : isNegative,
+              'ec-btn--sm' : isSmall,
+              'ec-btn--md' : isMedium,
+            }"
+            class="ec--m10 ec-btn"
+            :disabled="isDisabled"
+            >
+            <ec-icon v-if="hasIcon" name="simple-check" style="margin-right:10px;" size="20"/>
```

Comment:
There should be an error in the console complaining that `size` should be a number not a string. I think you need `:size="20"` everywhere.